src/openforms/conf/base.py

```diff
@@ -898,6 +898,7 @@
 API_VERSION = "2.5.0-alpha.0"
 
 SPECTACULAR_SETTINGS = {
+    "OAS_VERSION": "3.1.0",
```

Comment:
Let's do that in later in the future as part of Open Forms 3.0 - there are breaking changes in OAS 3.1, so that would require us to bump OF itself to a new major version.